src/blueprint.zig

```diff
@@ -335,6 +337,116 @@ pub const Pattern = struct {
 	}
 };
 
+pub const Mask = struct {
+	entries: ListUnmanaged(Entry),
+
+	pub const separator = ',';
+	pub const inverse = '!';
+	pub const tag = '$';
+	pub const property = '@';
```

Comment:
Should these be named? They are only used once, and it's clear from context what they are for.
(If you think they should be named, then they should definitely be private)

Reply:
Well, I think it is beneficial to have centralized place for declarations of reserved characters alongside their meaning encoded in variable names instead of comments.
I'll make them private.